tests/ui/test_capacity_breakdown_ui.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    skipif_ui_not_support,
+    bugzilla,
+)
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.ui.validation_ui import ValidationUI
+
+
+logger = logging.getLogger(__name__)
+
+
+class TestCapacityBreakdownUI(ManageTest):
+    """
+    Test Capacity Breakdown UI
+
+    Test Process:
+    1.Create new project
+    2.Create pvc with relevant sc [ceph-rbd,cephfs]
+    3.Create pod
+    4.Run FIO
+    5.Verify new project created on Capacity Breakdown UI
+    6.Verify new pod created on Capacity Breakdown UI
+
```

Comment:
Deletion of the Pod, PVC, namespace isn't mentioned. It will be good to add them in the sequence of deletion.